shims/test/src-2.13/peschke/UnitSpec.scala

```diff
@@ -0,0 +1,11 @@
+package peschke
+
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import org.scalatest.{EitherValues, OptionValues}
+
+trait UnitSpec
```

Comment:
Is this for compatibility with 2.13? If so what is the difference between the other `UnitSpec`?

Reply:
I wrote the tests in `src-2.13` because the test code uses `Iterator#nextOption`, probably could have written them in `src` using `next` and `hasNext`, but it seemed like too much of a hassle :)